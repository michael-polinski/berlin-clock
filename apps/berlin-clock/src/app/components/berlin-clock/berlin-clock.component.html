<berlin-clock-second-bar></berlin-clock-second-bar>
<berlin-clock-hours-bar></berlin-clock-hours-bar>
<berlin-clock-minutes-bar></berlin-clock-minutes-bar>
