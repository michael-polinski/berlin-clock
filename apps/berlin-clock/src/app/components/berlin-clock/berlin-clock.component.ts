import { Component } from '@angular/core';

/**
 * Component to display the 'Berlin-clock' (https://en.wikipedia.org/wiki/Mengenlehreuhr). This component is mostly
 * a wrapper for the different parts of the clock.
 */
@Component({
  selector: 'berlin-clock-berlin-clock',
  templateUrl: './berlin-clock.component.html',
  styleUrls: ['./berlin-clock.component.scss']
})
export class BerlinClockComponent {

}
