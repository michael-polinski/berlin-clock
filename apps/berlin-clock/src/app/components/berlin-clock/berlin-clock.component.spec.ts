import { ComponentFixture, TestBed } from '@angular/core/testing';

import { BerlinClockComponent } from './berlin-clock.component';

/**
 * TODO
 */
describe('BerlinClockComponent', () => {
  let component: BerlinClockComponent;
  let fixture: ComponentFixture<BerlinClockComponent>;

  beforeEach(async () => {
    await TestBed.configureTestingModule({
      declarations: [ BerlinClockComponent ]
    })
    .compileComponents();
  });

  beforeEach(() => {
    fixture = TestBed.createComponent(BerlinClockComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
  });

  it('should create', () => {
    expect(component).toBeTruthy();
  });
});
