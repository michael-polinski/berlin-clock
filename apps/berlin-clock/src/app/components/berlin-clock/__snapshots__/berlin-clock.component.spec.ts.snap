// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BerlinClockComponent test that the component should match its snapshot. 1`] = `
<berlin-clock-berlin-clock
  __ngContext__={[Function LRootView]}
>
  <div
    class="d-flex flex-column align-items-center"
  >
    <berlin-clock-second-bar>
      <berlin-clock-indicator>
        <div
          class="my-2 indicator circle-indicator"
        />
      </berlin-clock-indicator>
    </berlin-clock-second-bar>
    <berlin-clock-hours-bar>
      <div
        class="d-flex flex-column"
      >
        <div
          class="d-flex flex-row"
        >
          <berlin-clock-indicator>
            <div
              class="my-2 indicator rectangle-indicator"
            />
          </berlin-clock-indicator>
          <berlin-clock-indicator>
            <div
              class="my-2 indicator rectangle-indicator"
            />
          </berlin-clock-indicator>
          <berlin-clock-indicator>
            <div
              class="my-2 indicator rectangle-indicator"
            />
          </berlin-clock-indicator>
          <berlin-clock-indicator>
            <div
              class="my-2 indicator rectangle-indicator"
            />
          </berlin-clock-indicator>
        </div>
        <div
          class="d-flex flex-row"
        >
          <berlin-clock-indicator>
            <div
              class="my-2 indicator rectangle-indicator"
            />
          </berlin-clock-indicator>
          <berlin-clock-indicator>
            <div
              class="my-2 indicator rectangle-indicator"
            />
          </berlin-clock-indicator>
          <berlin-clock-indicator>
            <div
              class="my-2 indicator rectangle-indicator"
            />
          </berlin-clock-indicator>
          <berlin-clock-indicator>
            <div
              class="my-2 indicator rectangle-indicator"
            />
          </berlin-clock-indicator>
        </div>
      </div>
    </berlin-clock-hours-bar>
    <berlin-clock-minutes-bar>
      <div
        class="d-flex flex-column"
      >
        <div
          class="d-flex flex-row"
        >
          <berlin-clock-indicator>
            <div
              class="my-2 indicator rectangle-indicator small-rectangle"
            />
          </berlin-clock-indicator>
          <berlin-clock-indicator>
            <div
              class="my-2 indicator rectangle-indicator small-rectangle"
            />
          </berlin-clock-indicator>
          <berlin-clock-indicator>
            <div
              class="my-2 indicator rectangle-indicator small-rectangle"
            />
          </berlin-clock-indicator>
          <berlin-clock-indicator>
            <div
              class="my-2 indicator rectangle-indicator small-rectangle"
            />
          </berlin-clock-indicator>
          <berlin-clock-indicator>
            <div
              class="my-2 indicator rectangle-indicator small-rectangle"
            />
          </berlin-clock-indicator>
          <berlin-clock-indicator>
            <div
              class="my-2 indicator rectangle-indicator small-rectangle"
            />
          </berlin-clock-indicator>
          <berlin-clock-indicator>
            <div
              class="my-2 indicator rectangle-indicator small-rectangle"
            />
          </berlin-clock-indicator>
          <berlin-clock-indicator>
            <div
              class="my-2 indicator rectangle-indicator small-rectangle"
            />
          </berlin-clock-indicator>
          <berlin-clock-indicator>
            <div
              class="my-2 indicator rectangle-indicator small-rectangle"
            />
          </berlin-clock-indicator>
          <berlin-clock-indicator>
            <div
              class="my-2 indicator rectangle-indicator small-rectangle"
            />
          </berlin-clock-indicator>
          <berlin-clock-indicator>
            <div
              class="my-2 indicator rectangle-indicator small-rectangle"
            />
          </berlin-clock-indicator>
        </div>
        <div
          class="d-flex flex-row"
        >
          <berlin-clock-indicator>
            <div
              class="my-2 indicator rectangle-indicator"
            />
          </berlin-clock-indicator>
          <berlin-clock-indicator>
            <div
              class="my-2 indicator rectangle-indicator"
            />
          </berlin-clock-indicator>
          <berlin-clock-indicator>
            <div
              class="my-2 indicator rectangle-indicator"
            />
          </berlin-clock-indicator>
          <berlin-clock-indicator>
            <div
              class="my-2 indicator rectangle-indicator"
            />
          </berlin-clock-indicator>
        </div>
      </div>
    </berlin-clock-minutes-bar>
  </div>
</berlin-clock-berlin-clock>
`;
